index.html

```diff
@@ -112,29 +110,26 @@ <h2>
       </h2>
       <p>
         New types of mobile devices are appearing that have some sort of folding capabilities around the screen. Hence, 
-        when the device folds, it physically forms an angle. The main interest in knowing the fold angle is because there
-         are opportunities in responsive design that enable new user experiences.
+        when the device folds, it physically can be different form factor. The main interest in knowing the posture is because there
+        are opportunities in responsive design that enable new user experiences.
```

Comment:
```suggestion
        when the device folds, it can be a physically different form factor. The main interest in knowing the posture is to enable new user experiences with responsive design.
```